### PR TITLE
Add Status to Anime

### PR DESCRIPTION
**Pull request by auhc99**
Multiple changes are made:
- Status Attribute is added
- Update Status Command is added

Currently still missing:
- [x] Missing Testcases
- [x] Changes to UI
- [x] Merge conflicts with ryan's pr
- [x] Changes not saved 
- [x] Fix editCommand test case / Remove editCommand

DONT MERGE TILL ALL ARE COMPLETED

### CODE REVIEW

File: src/main/java/seedu/anilist/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -83,9 +85,11 @@ private static Anime createEditedAnime(Anime animeToEdit, EditAnimeDescriptor ed
         assert animeToEdit != null;
 
         Name updatedName = editAnimeDescriptor.getName().orElse(animeToEdit.getName());
+        Episode episode = animeToEdit.getEpisode();
+        Status status = animeToEdit.getStatus();
```
Comment (muhammad-faruq) [inline src/main/java/seedu/anilist/logic/commands/EditCommand.java:89]: This one dont need optional unlike the rest? Why is it that the rest of them is using optionals?

File: src/test/java/seedu/anilist/testutil/AnimeBuilder.java [hunk 2]
```diff
@@ -15,17 +16,21 @@
 public class AnimeBuilder {
 
     public static final String DEFAULT_NAME = "Amy Bee";
```
Comment (muhammad-faruq) [inline src/test/java/seedu/anilist/testutil/AnimeBuilder.java:18]: can change the name of Amy Bee to the most generic anime name that you can think of

File: src/test/java/seedu/anilist/testutil/TypicalAnimes.java [hunk 3]
```diff
@@ -17,15 +17,39 @@
...
+            .build();
+    public static final Anime FIONA = new AnimeBuilder()
+            .withName("Fiona Kunz")
+            .withEpisode("24")
+            .withStatus("F")
+            .build();
+    public static final Anime GEORGE = new AnimeBuilder()
+            .withName("George Best")
+            .withStatus("T")
+            .build();
```
Comment (muhammad-faruq) [inline src/test/java/seedu/anilist/testutil/TypicalAnimes.java:53]: If got time can change the names of these people to anime names

File: src/test/java/seedu/anilist/testutil/AnimeBuilder.java (hunk 2 above)
Comment (luodan01) [reply src/test/java/seedu/anilist/testutil/AnimeBuilder.java:18]: @muhammad-faruq I've changed it in my pr

File: src/main/java/seedu/anilist/logic/commands/EditCommand.java (hunk 1 above)
Comment (muhammad-faruq) [reply src/main/java/seedu/anilist/logic/commands/EditCommand.java:89]: okie coolios

**Review by muhammad-faruq — APPROVED** [review]
lgtm

**Pull request merged**